cmd/protoc-gen-connect-go/main.go

```diff
@@ -283,17 +268,19 @@ func generateClientImplementation(g *protogen.GeneratedFile, service *protogen.S
 		)
 		g.P("httpClient,")
 		g.P(`baseURL + `, procedureConstName(method), `,`)
+		g.P(connectPackage.Ident("WithSchema"), "(",
+			g.QualifiedGoIdent(file.GoDescriptorIdent),
```

Comment:
Should I break out the service as a variable?

Reply:
Have moved to a `serviceDescriptor` within the func. 

Adding a private global var for each method that resolves the method descriptor would move the panic to the init. Shall I try it out? Panicing on the `NewClient` or `NewHandler` call over global vars. Would there be any use cases for someone modifying the public file descriptor var and then calling `NewClient` and expecting the types to include the modified descriptors?

Reply:
> Adding a private global var for each method that resolves the method descriptor would move the panic to the init. Shall I try it out?

Yeah, I think that would be better.

> Would there be any use cases for someone modifying the public file descriptor var and then calling NewClient and expecting the types to include the modified descriptors?

Eek, that sounds like a seriously misbehaving program, and I personally don't think we should bother trying to support that case.

Reply:
Switched to global var descriptors.